dev-python/dbuspython/dbuspython-1.2.14.recipe

```diff
@@ -0,0 +1,110 @@
+SUMMARY="The original dbus implementation for Python"
+DESCRIPTION="dbus-python is the original Python binding \
+for dbus, the reference implementation of the D-Bus protocol."
+HOMEPAGE="https://www.freedesktop.org/wiki/Software/DBusBindings/"
+COPYRIGHT="2002-2010 Red Hat Inc.
+	2003 David Zeuthen
+	2002-2003 CodeFactory AB
+	2003 James Willcox
+	2004 Rob Taylor
+	2004 Anders Carlsson
+	2005 Colin Walters
+	2006 Steve Frécinaux
+	2009-2011 Nokia Corporation
+	2010-2012 Mike Gorse
+	2016 Simon McVittie
+	2005-2019 Collabora Ltd."
+LICENSE="MIT
+	AFL v2.1
+	GNU GPL v2"
+REVISION="1"
+SOURCE_URI="https://dbus.freedesktop.org/releases/dbus-python/dbus-python-$portVersion.tar.gz"
+CHECKSUM_SHA256="b10206ba3dd641e4e46411ab91471c88e0eec1749860e4285193ee68df84ac31"
+SOURCE_DIR="dbus-python-$portVersion"
+
+ARCHITECTURES="!x86_gcc2 x86_64"
+SECONDARY_ARCHITECTURES="x86"
+
+PROVIDES="
+	dbuspython$secondaryArchSuffix = $portVersion
+	"
+REQUIRES="
+	haiku$secondaryArchSuffix
```

Comment:
I'm actually kinda torn between `haiku` or `haiku_devel` here, since these are headers. Any suggestions @Begasus?

Reply:
Check for usage of "haiku_devel" in REQUIRES, haven't seen it used here before?

Reply:
I guess it's fine as is.